app/components/collections/edit/workflow_component.html.erb

```diff
@@ -14,6 +14,6 @@
   <% end %>
 
   <%= tag.div data: { workflow_target: 'reviewersSection' }, class: 'd-none' do %>
-    <%= render Collections::Edit::BulkParticipantsComponent.new(form:, field_name: :reviewers, model_class: ReviewerForm) %>
+    <%= render Edit::BulkParticipantsComponent.new(form:, field_name: :reviewers, model_class: ReviewerForm, form_component: Collections::Edit::ParticipantComponent) %>
```

Comment:
The existing `Collections::Edit::BulkParticipantsComponent` was moved and tweaked so that it could be used for editing shares.